components/src/core/AppBar/AppBar.tsx

```diff
@@ -287,6 +287,7 @@ const AppBarRender: React.ForwardRefRenderFunction<unknown, AppBarProps> = (prop
         <Root
             ref={ref}
             {...muiAppBarProps}
+            data-testid={'appbar-root'}
```

Comment:
I'm thinking these `data-testid` tags need to be prefixed with `blui-` in the event that there's a duplicated test id in our adopters' apps.

Reply:
by default react test library looks for data-testid and changing them to be blui-data-testid or data-blui-testid breaks the findByTestId methods. I'm sure there are config options for this but would be a hack